### PR TITLE
Update milestone information in genesis file

### LINKED ISSUE #972: Recommend using latest milestones in private networks
Recommend using later milestones, and maybe outline the relationship between milestones, opcodes and hardforks

eg this page has a genesis file with only constantinople defined, but if users need later milestones/opcodes this is not mentioned

https://besu.hyperledger.org/en/stable/HowTo/Configure/Consensus-Protocols/Clique/

Milestones are mentioned here https://besu.hyperledger.org/en/stable/Reference/Config-Items/#milestone-blocks - maybe make this more visible.

Has caused confusion eg https://github.com/hyperledger/besu/issues/3549

- [ ] Missing content
- [x] Outdated content
- [ ] Wrong content
- [ ] Confusing or misleading content
- [ ] Other

## The broken page
https://besu.hyperledger.org/en/stable/HowTo/Configure/Consensus-Protocols/Clique/

### PR DESCRIPTION
**Pull request by rolandtyler**
Signed-off-by: Roland Tyler <roland.tyler@consensys.net>

## Pull request checklist

Use the following list to make sure your PR fits the Besu doc quality standard.

### Before creating the pull request

Make sure that:

- [x] [all commits in this PR are signed off for the DCO](https://wiki.hyperledger.org/display/BESU/DCO).
- [x] you read the [contribution guidelines](https://wiki.hyperledger.org/display/BESU/Contributing+to+documentation).
- [x] you have [tested your changes locally](https://wiki.hyperledger.org/display/BESU/MkDocs+And+Markdown+Guide#MkDocsAndMarkdownGuide-PreviewTheDocumentation) before submitting them to the community for review.

### After creating your pull request and tests finished

Make sure that:

- [x] you fixed all the issues raised by the tests, if any.
- [x] you verified the rendering of your changes on [ReadTheDocs.org PR preview](https://wiki.hyperledger.org/display/BESU/MkDocs+And+Markdown+Guide#MkDocsAndMarkdownGuide-PreviewwithReadTheDocs)
  and updated the testing link (see [Testing](#testing)).

## Describe the change
Update milestone examples
Add recommendation to use latest milestone for private networks.

## Issue fixed
Fixes #972 

### For content changes

- [x] Doc content
- [ ] Doc pages organisation

### For tools changes

- [ ] CircleCI workflow
- [ ] Build and QA tools (lint, vale,…)
- [ ] MkDocs templates
- [ ] MkDocs configuration
- [ ] Python dependencies
- [ ] Node dependencies and JavaScript
- [ ] ReadTheDocs configuration
- [ ] GitHub integration

### CODE REVIEW

File: docs/Tutorials/Private-Network/Create-Private-Network.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ the `Private-Network` directory:
 ```json
 {
   "config": {
-      "constantinoplefixblock": 0,
+    "londonBlock": 0,
```
Comment (macfarla) [inline docs/Tutorials/Private-Network/Create-Private-Network.md:61]: if you're adjusting the indent here, the following line (ethash) should also move

File: docs/Reference/Config-Items.md [hunk 2]
```diff
@@ -83,16 +83,17 @@ In private networks, the milestone block defines the protocol version for the ne
...
+        "londonBlock": 0,
         ...
       },
     }
     ```
 
 !!! tip
 
-    When specifying the milestone block for private networks, you only need to specify the latest
+    When specifying the milestone block for private networks, we recommend specifying the latest
```
Comment (macfarla) [inline docs/Reference/Config-Items.md:94]: We recommend specifying the latest milestone block for private networks.
or
When starting a new private network, we recommend specifying the latest milestone

**Review by macfarla — APPROVED** [review]
couple of small comments

File: docs/Tutorials/Private-Network/Create-Private-Network.md (hunk 1 above)
Comment (rolandtyler) [reply docs/Tutorials/Private-Network/Create-Private-Network.md:61]: addressed

File: docs/Reference/Config-Items.md (hunk 2 above)
Comment (rolandtyler) [reply docs/Reference/Config-Items.md]: addressed

File: docs/Reference/Config-Items.md [hunk 3]
```diff
@@ -83,16 +83,17 @@ In private networks, the milestone block defines the protocol version for the ne
...
       "config": {
         ...
-        "constantinopleFixBlock": 0,
+        "londonBlock": 0,
         ...
       },
     }
     ```
 
 !!! tip
```
Comment (alexandratran) [inline docs/Reference/Config-Items.md:92]:
```suggestion
!!! note
```

File: docs/Reference/Config-Items.md [hunk 4]
```diff
@@ -83,16 +83,17 @@ In private networks, the milestone block defines the protocol version for the ne
...
 
-    When specifying the milestone block for private networks, you only need to specify the latest
-    milestone. It is implied this includes the preceding milestones.
+    We recommend specifying the latest milestone block for private networks.
+    It is implied this includes the preceding milestones.
+    Specifying the latest milestone ensures you are using the most up-to-date protocol and have access to the most recent opcodes.
```
Comment (alexandratran) [inline docs/Reference/Config-Items.md:96]:
```suggestion
    This ensures you're using the most up-to-date protocol and have access to the most recent opcodes.
```

**Review by alexandratran — APPROVED** [review]
Suggestion.

File: docs/Reference/Config-Items.md (hunk 3 above)
Comment (rolandtyler) [reply docs/Reference/Config-Items.md]: addressed

File: docs/Reference/Config-Items.md (hunk 4 above)
Comment (rolandtyler) [reply docs/Reference/Config-Items.md]: addressed

**Pull request merged**